### PR TITLE
zombie powder fix

### PR DESCRIPTION
**Pull request by carlarctg**
# About the pull request

Fixes Zombie Powder bugging the fuck out by slapping a ton of FAKEDEATH checks everywhere. It now properly shows the holder as dead on HUDs and scans.

Fixed an issue in which sometimes qdeleted reagents still procced on life.

Fixed examining things changing your direction if you're incapacitated.

Added FAKEDEATH to the mob_incapacitated() check.

# Explain why it's good for the game

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
fix: Fixes Zombie Powder bugging the fuck out by slapping a ton of FAKEDEATH checks everywhere. It now properly shows the holder as dead on HUDs and scans.
fix: Fixed an issue in which sometimes qdeleted reagents still procced on life.
fix: Fixed examining things changing your direction if you're incapacitated.
fix: Added FAKEDEATH to the mob_incapacitated() check.
/:cl:

### CODE REVIEW

**Comment by Huffie56** [comment]
will it fix that resurrect timer bug?https://github.com/cmss13-devs/cmss13/issues/2267

**Review by harryob — CHANGES_REQUESTED** [review]
you automatically check the pulse of really dead people, but not fakedead, because it checks for just stat atm

https://github.com/carlarctg/cmss13/blob/983f7fdbfeb20b243a07667fbb5e37d7b2a3fac2/code/modules/mob/living/carbon/human/examine.dm#L197

clicky

**Review by harryob — CHANGES_REQUESTED** [review]
change the if(stat) to if(stat == DEAD | |status_flags & FAKEDEATH) in the link above

File: code/modules/mob/living/living_healthscan.dm [hunk 1]
```diff
@@ -60,15 +60,36 @@ GLOBAL_LIST_INIT(known_implants, subtypesof(/obj/item/implant))
 		ui.open()
 		ui.set_autoupdate(FALSE)
 
+/datum/health_scan/proc/get_death_value(mob/target_mob)
```
Comment (Zonespace27) [inline code/modules/mob/living/living_healthscan.dm:63]: code doc

File: code/modules/mob/living/living_healthscan.dm [hunk 2]
```diff
@@ -60,15 +60,36 @@ GLOBAL_LIST_INIT(known_implants, subtypesof(/obj/item/implant))
...
+	if(target_mob.stat == DEAD || target_mob.status_flags & FAKEDEATH)
+		return TRUE
+	else
```
Comment (Zonespace27) [inline code/modules/mob/living/living_healthscan.dm:66]: else not needed

File: code/modules/mob/living/living_healthscan.dm [hunk 3]
```diff
@@ -60,15 +60,36 @@ GLOBAL_LIST_INIT(known_implants, subtypesof(/obj/item/implant))
...
+		return FALSE
+
+/datum/health_scan/proc/get_oxy_value(mob/target_mob)
```
Comment (Zonespace27) [inline code/modules/mob/living/living_healthscan.dm:69]: codedoc

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

**Pull request merged**